src/Query/Builder.php

```diff
@@ -556,6 +557,8 @@ public function generateCacheKey()
     /** @return ($function is null ? AggregationBuilder : mixed) */
     public function aggregate($function = null, $columns = ['*'])
     {
+        assert(is_array($columns), new TypeError(sprintf('Argument #2 ($columns) must be of type array, %s given', get_debug_type($columns))));
```

Comment:
We cannot add the type to the argument as it's missing in the parent method. When the value is invalid here, the error comes very late in `Query\Builder::toMQL`.